Add NE mgmt track report type

sbreitbart-NOAA opened the pull request:
<!---
Thanks for opening a PR. This commented text will **NOT** appear in the final PR. Toggle between Write and Preview to see what your PR will look like without the comments.
-->

# What is the feature?
* Option to create a report with a NE mgmt track template

# How have you implemented the solution?
* New folder in inst with template files

# Does the PR impact any other area of the project, maybe another repo?
* No

sbreitbart-NOAA commented:
Note to @Schiano-NOAA: I made this PR now, before formally reviewing yours, because Kiva asked for a couple weeks to review the pfmc outline but we can aim to merge in this new architecture, and NE mgmt track template, before then.

Schiano-NOAA commented:
> Note to @Schiano-NOAA: I made this PR now, before formally reviewing yours, because Kiva asked for a couple weeks to review the pfmc outline but we can aim to merge in this new architecture, and NE mgmt track template, before then.

I actually made the changes to create_template on the other branch, but I don't think there's a rush on this so we can just merge into that branch after.

Did you create this branch off of the one where I created the pfmc outline or from dev? If you created from dev, could you switch it to the one I created? I intended the merge to go like this:

new-template-set-up --------------ne-temp-----------------new-template-set-up-------------dev
new-template-set-up----------safe-template-------------------------^

sbreitbart-NOAA commented:
> > Note to @Schiano-NOAA: I made this PR now, before formally reviewing yours, because Kiva asked for a couple weeks to review the pfmc outline but we can aim to merge in this new architecture, and NE mgmt track template, before then.
> 
> I actually made the changes to create_template on the other branch, but I don't think there's a rush on this so we can just merge into that branch after.
> 
> Did you create this branch off of the one where I created the pfmc outline or from dev? If you created from dev, could you switch it to the one I created? I intended the merge to go like this:
> 
> new-template-set-up --------------ne-temp-----------------new-template-set-up-------------dev new-template-set-up----------safe-template-------------------------^
> 
> In this way, we can have the final commit message merging into dev as just "add new type templates" or what not

Just rebased and changed the base branch 👍

Schiano-NOAA commented on inst/templates/nemt/01_executive_summary.qmd:
I was thinking since there isn't much that consitutes this skeleton, that we could split up the sections after the executive summary and place them into their own quarto files. This was it becomes more modular. I would also move the subsections as their own sections with the same header level as executive summary

sbreitbart-NOAA replied:
Sure, I can do that. What should I do with the pasted in TORs, then? I'm not sure how to split it among the sections. Just remove it?

Schiano-NOAA replied:
Actually, after having some time to think it over, keep the outline as is but add a new quarto file after the executive summary called "reviewer_comments". Thanks!
